pkg/pmem-csi-driver/controllerserver-master.go

```diff
@@ -435,3 +435,7 @@ func (cs *masterController) getVolumeByName(Name string) *pmemVolume {
 	}
 	return nil
 }
+
+func (cs *masterController) ControllerExpandVolume(context.Context, *csi.ControllerExpandVolumeRequest) (*csi.ControllerExpandVolumeResponse, error) {
```

Comment:
@pohly, can you move this default implementation to `controllerserver-default.go`, so that both master and node inherits them.

Reply:
Oops, I realized that these changes are already merged. So Ignore my comment.